pkg/api/validation/validation.go

```diff
@@ -397,6 +401,20 @@ func validateISCSIVolumeSource(iscsi *api.ISCSIVolumeSource) errs.ValidationErro
 	return allErrs
 }
 
+func validateFCVolumeSource(fc *api.FCVolumeSource) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+	if len(fc.TargetWWNs) < 1 {
+		allErrs = append(allErrs, errs.NewFieldRequired("targetWWNs"))
+	}
+	if fc.FSType == "" {
+		allErrs = append(allErrs, errs.NewFieldRequired("fsType"))
+	}
+	if fc.Lun < 0 || fc.Lun > 255 {
```

Comment:
Because if empty in JSON/YAML it will default to 0.  0 passes your validation, even though it may not be correct.

If you default to -1, then the user is forced to enter a real value (which may be 0, but at least it's on purpose)